Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGppDataStore.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using BuildXL.Engine.Cache.KeyValueStores;
+using BuildXL.Execution.Analyzer;
+using BuildXL.Scheduler.Tracing;
+using BuildXL.Utilities;
+using Google.Protobuf;
+
+namespace BuildXL.Analyzers.Core.XLGPlusPlus
+{
+    public class XLGppDataStore
+    {
+        /// <summary>
+        /// Rocks DB Accessor for XLG++ data
+        /// </summary>
+        public KeyValueStoreAccessor Accessor { get; set; }

```

Comment:
>set [](start = 53, length = 3)

should we make this private?

Reply:
I was thinking about that as well, but I wasn't sure if we wanted writing logic in this class or just in the Analyzer (in which case, the analyzer needs the Accessor to be public so it can write into the data store). 

---
In reply to: [306046558](https://github.com/microsoft/BuildXL/pull/636#discussion_r306046558) [](ancestors = 306046558)